packages/wallets/src/features/cashier/flows/WalletWithdrawal/WalletWithdrawal.tsx

```diff
@@ -22,13 +22,20 @@ const WalletWithdrawal = () => {
         const queryParams = new URLSearchParams(location.search);
         const loginidQueryParam = queryParams.get('loginid');
         const verificationQueryParam = queryParams.get('verification');
+        const localVerificationCode = localStorage.getItem('verification_code.payment_withdraw');
 
         // if loginid query param doesn't match active wallet's loginid on mount, initiate account switching
         if (loginidQueryParam && loginidQueryParam !== activeWallet?.loginid) {
             switchAccount(loginidQueryParam);
             return;
         }
 
+        // for third party redirections where verification code is fetched from local storage value saved in client store
+        if (localVerificationCode) {
+            setVerificationCode(localVerificationCode);
+            localStorage.removeItem('verification_code.payment_withdraw');
+        }
```

Comment:
Different approach used for third party redirection as redirect url has a different format and does not pass through the `redirect.jsx` file in core before the parameters in the url gets removed.

Initial code received from the PTA token generated url gets stored in local storage. This value then gets used and removed in the wallets withdrawal module

<img width="1205" alt="Screenshot 2024-06-26 at 2 20 09 PM" src="https://github.com/binary-com/deriv-app/assets/142860499/a80cae5b-c61d-43db-9b4d-bdbc388f0eba">

Reply:
PTA token method for withdrawal verification:

https://github.com/binary-com/deriv-app/assets/142860499/b9d1c9cc-7d18-4fdc-b037-5d2e7443c98c